tests/unit/sagemaker/jumpstart/test_utils.py

```diff
@@ -112,3 +121,168 @@ def test_get_sagemaker_version(patched_parse_sm_version: Mock):
+        tags=tags,
+        inference_model_uri=inference_model_uri,
+        inference_script_uri=inference_script_uri,
+    ) == [
+        {"some": "tag"},
+        {JumpStartTag.INFERENCE_MODEL_URI.value: inference_model_uri},
+    ]
+
+    tags = None
+    inference_script_uri = random_jumpstart_s3_uri("random_key")
+    inference_model_uri = "dfsdfs"
+    assert (
+        utils.add_jumpstart_tags(
+            tags=tags,
+            inference_model_uri=inference_model_uri,
+            inference_script_uri=inference_script_uri,
+        )
+        == [{JumpStartTag.INFERENCE_SCRIPT_URI.value: inference_script_uri}]
+    )
+
+    tags = []
+    inference_script_uri = random_jumpstart_s3_uri("random_key")
+    inference_model_uri = "dfsdfs"
+    assert (
+        utils.add_jumpstart_tags(
+            tags=tags,
+            inference_model_uri=inference_model_uri,
+            inference_script_uri=inference_script_uri,
+        )
+        == [{JumpStartTag.INFERENCE_SCRIPT_URI.value: inference_script_uri}]
+    )
+
+    tags = [{"some": "tag"}]
+    inference_script_uri = random_jumpstart_s3_uri("random_key")
+    inference_model_uri = "dfsdfs"
+    assert utils.add_jumpstart_tags(
+        tags=tags,
+        inference_model_uri=inference_model_uri,
+        inference_script_uri=inference_script_uri,
+    ) == [
+        {"some": "tag"},
+        {JumpStartTag.INFERENCE_SCRIPT_URI.value: inference_script_uri},
+    ]
+
+    tags = None
+    inference_script_uri = random_jumpstart_s3_uri("random_key")
+    inference_model_uri = random_jumpstart_s3_uri("random_key")
+    assert utils.add_jumpstart_tags(
+        tags=tags,
+        inference_model_uri=inference_model_uri,
+        inference_script_uri=inference_script_uri,
+    ) == [
+        {
+            JumpStartTag.INFERENCE_MODEL_URI.value: inference_model_uri,
+        },
+        {JumpStartTag.INFERENCE_SCRIPT_URI.value: inference_script_uri},
+    ]
+
+    tags = []
+    inference_script_uri = random_jumpstart_s3_uri("random_key")
+    inference_model_uri = random_jumpstart_s3_uri("random_key")
+    assert utils.add_jumpstart_tags(
+        tags=tags,
+        inference_model_uri=inference_model_uri,
+        inference_script_uri=inference_script_uri,
+    ) == [
+        {
+            JumpStartTag.INFERENCE_MODEL_URI.value: inference_model_uri,
+        },
+        {JumpStartTag.INFERENCE_SCRIPT_URI.value: inference_script_uri},
+    ]
+
+    tags = [{"some": "tag"}]
+    inference_script_uri = random_jumpstart_s3_uri("random_key")
+    inference_model_uri = random_jumpstart_s3_uri("random_key")
+    assert utils.add_jumpstart_tags(
+        tags=tags,
+        inference_model_uri=inference_model_uri,
+        inference_script_uri=inference_script_uri,
+    ) == [
+        {"some": "tag"},
+        {
+            JumpStartTag.INFERENCE_MODEL_URI.value: inference_model_uri,
+        },
+        {JumpStartTag.INFERENCE_SCRIPT_URI.value: inference_script_uri},
+    ]
```

Comment:
could you please add a unit test for the edge case where these tag keys are requested by the customer?